cmd/geth/main.go

```diff
@@ -417,6 +428,35 @@ func attach(ctx *cli.Context) {
 	}
 }
 
+func initGenesis(ctx *cli.Context) {
+	genesisPath := ctx.Args().First()
+	if len(genesisPath) == 0 {
+		utils.Fatalf("must supply path to genesis JSON file")
+	}
+
+	var (
+		datadir = utils.MustMakeDataDir(ctx)
+		cache   = ctx.GlobalInt(utils.CacheFlag.Name)
+		handles = utils.MakeDatabaseHandles()
```

Comment:
You don't really need `cache` and `handles` here. If you pass in 0 for both it will default to 16MB cache and 16 handles, neither of which will make much of a difference for only writing a couple values into the database.